kinetic/distribution.yaml

```diff
@@ -7586,10 +7586,6 @@ repositories:
       type: git
       url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
       version: master
-    source:
```

Comment:
If you want to disable CI please keep the source entry but turn off ci using `test_commits: false` like https://github.com/ros/rosdistro/blob/bfaaa5c587d42900befecda8b4cbb983028ce7af/kinetic/distribution.yaml#L2496 instead of removing the whole entry. The source entry allows it to be automatically generated into source checkouts.